api/axfeat/Cargo.toml

```diff
@@ -13,7 +13,7 @@ documentation = "https://arceos-org.github.io/arceos/axfeat/index.html"
 default = []
 
 # Multicore
-smp = ["axhal/smp", "axruntime/smp", "kspin/smp"]
+smp = ["axhal/smp", "axruntime/smp", "axtask/smp", "kspin/smp"]
```

Comment:
`"kspin/smp"` can be removed, it has been added in `axtask/Cargo.toml`

Reply:
Sorry, we cannot remove `kspin/smp`, consider the `smp` feature is on and the `multitask` feature is off, we should enable `smp` in `kspin`.

Reply:
yep